docs/TEAMS.md

```diff
@@ -0,0 +1,109 @@
+Organic Maps is led by community contributors, with functional domains occasionally establishing or dissolving their leadership structures, formal or informal. 
+
+This file outlines the functional project structure that has evolved over time. The current list of teams is available on [GitHub](https://github.com/orgs/organicmaps/teams), accessible only to [GitHub Org members](https://github.com/orgs/organicmaps/people).
+
+Below is the list of teams and groups to tag on GitHub if you need assistance in a specific area. If you are unsure which team to contact, please tag  [@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage) for help.
+
+## Contributors
+
+[@organicmaps/contributors](https://github.com/orgs/organicmaps/teams/contributors/teams)
+
+Active code contributors who can create branches and manage issues in the main repo. See [subteams](https://github.com/orgs/organicmaps/teams/contributors/teams).
+
+### Triage
+
+[@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage)
+
+A special subteam to triage incoming issues and pull requests. Please tag this group if you're unsure which group to tag.
+
+## Maintainers
+
+[@organicmaps/maintainers](https://github.com/orgs/organicmaps/teams/maintainers)
+
+People who can merge PRs.
+
+## Android
+
+[@organicmaps/android](https://github.com/orgs/organicmaps/teams/android/members)
+
+Android contributors,
+
+### Android Auto
+
+Android Auto contributors (a subteam of Android).
+
+[@organicmaps/android-auto](https://github.com/orgs/organicmaps/teams/android-auto/members)
+
+## iOS
+
+[@organicmaps/ios](https://github.com/orgs/organicmaps/teams/core/members)
+
+iOS contributors.
+
+## C++
+
+[@organicmaps/cpp](https://github.com/orgs/organicmaps/teams/cpp/members)
+
+C++ contributors.
```

Comment:
It makes sense to add a Python team as well.

Reply:
Python is too broad and can be used in many different ways. I’m not a fan of the current C++ team name, and I think it would make more sense to name it based on the domain. The same goes for Python. Instead of naming teams after specific tools, let’s have teams for specific areas like "generator" or "translation tools."

Does it make sense to decompose C++/Core into subteams already? For example, for rending we already have a dedicated expert..

Reply:
decomposing would be convenient, but we need a broader "app core" team anyway I think